persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/MolecularDataMyBatisRepositoryTest.java

```diff
@@ -44,12 +44,12 @@ public void getGeneMolecularAlterations() throws Exception {
         GeneMolecularAlteration molecularAlteration1 = result.get(0);
         Assert.assertEquals((Integer) 207, molecularAlteration1.getEntrezGeneId());
         String[] expected = {"-0.4674","-0.6270","-1.2266","-1.2479","-1.2262","0.6962","-0.3338","-0.1264","0.7559","-1.1267","-0.5893",
-        "-1.1546","-1.0027","-1.3157"};
+        "-1.1546","-1.0027","-1.3157",""};
```

Comment:
This is kinda weird but it seems that the imported testSQL has a trailing `,` for all these datasets